### PR TITLE
Added code to handle accusations

### PR DESCRIPTION
**Pull request by kcavicchio**

### CODE REVIEW

**Review by EthanChe — APPROVED** [review]
lgtm!

File: src/game/Player.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ class Player extends schema.Schema{
       this.currentLocation = "";
       this.startX = x;
       this.startY = y;
+      this.isActive = true;
```
Comment (EthanChe) [inline src/game/Player.js:14]: If we want this variable to be part of the state, its type needs to be defined.

**Review by EthanChe — CHANGES_REQUESTED** [review]

**Review by zlmcdaniel — APPROVED** [review]
Looks good!

**Review by EthanChe — APPROVED** [review]
lgtm

**Pull request merged**